app/build.gradle

```diff
@@ -26,11 +26,11 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
     }
     kotlinOptions {
-        jvmTarget = '1.8'
+        jvmTarget = '11'
```

Comment:
Replace Main view to text label "you are loh" and then application must exit after 5 seconds of boot

Reply:
Replace application to ONE button "you are loh" and then application must exit after 5 seconds of boot